package.json

```diff
@@ -1,14 +1,16 @@
 {
-    "name": "Underfitted Membership NFT",
+    "name": "underfitted-membership-nft",
     "version": "0.1.0",
     "description": "Smart contract for the Underfitted Membership NFT token",
     "scripts": {
         "build": "hardhat compile",
         "test": "hardhat test",
         "gas": "REPORT_GAS=1 hardhat test",
         "coverage": "hardhat coverage",
-        "deploy:polygon": "npx hardhat run scripts/deploy.js --network polygon",
-        "deploy:mumbai": "npx hardhat run scripts/deploy.js --network mumbai",
+        "deployNFT:polygon": "npx hardhat run scripts/deployNFT.js --network polygon",
```

Comment:
I think a pattern like `deploy:nft:polygon` and `deploy:token:mumbai` is a bit cleaner